python/src/trezorlib/cli/debug.py

```
@@ -118,23 +135,17 @@ def prodtest_t1(session: "Session") -> None:
-    debug_transport.open()
-    debug = DebugLink(transport=debug_transport)
-    debuglink_optiga_set_sec_max(debug)
-    debug_transport.close()
+    with debuglink_context(session.client.transport) as debug:
+        debuglink_optiga_set_sec_max(debug)
 
 
 @cli.command()
 @click.argument("filter", required=False)
 @with_session(seedless=True)
 def set_log_filter(session: "Session", filter: str) -> None:
```

Ruff (0.15.20)

[error] Function argument `filter` is shadowing a Python builtin

(A002)